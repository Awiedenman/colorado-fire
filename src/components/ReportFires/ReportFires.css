h3 {
  font-size: 1rem;
}

.reportFires-form {
  display: flex;
  align-items: center;
  justify-content: center;
  width: 40vw;
  height: 80vh;
}

.reportFire-form-section {
  text-align: center;
  padding: 10px;
  margin: 25px;
}

form {
  position: absolute;
  left: 0;
  top: 9.5rem;
  text-align: center;
  background-color: rgba(250, 250, 250, 0.806);
  height: 100vh;
  width: 18rem;
  overflow-y: scroll;
  border-right: 2px solid rgba(153, 149, 149, 0.835);
}

button {
  width: 5rem;
  border: 1px #000 ridge;
  border-radius: .2rem;
  box-shadow: 1px 1px 1px
}

button:hover {
  background-color: aquamarine;
}

input {
  border: none;
  border-bottom: 1px solid #000;
  margin: 5px;
  background-color: transparent;
  color: #232323;
  font-size: 1rem;
}

input:hover {
  border: none;
  border-bottom: 3px solid #052709;
  margin-bottom: 3px;
}

input::placeholder {
  color: #232323;
}

h3 {
  padding: 10px;
  color: #052709;
}