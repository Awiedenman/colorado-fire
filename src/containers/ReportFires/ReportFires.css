.reportFires {
  display: flex;
}

.reportFires-form {
  display: inline-block;
  width: 40vw;
}

.reportFires-map {
  display: inline-block;
  width: 40vw;
}

.reportFire-form-section {
  text-align: center;
  outline: 1px #000 ridge;
  padding: 10px;
  margin: 25px;
}

form {
  text-align: center;
  outline: 1px #000 ridge;
  padding: 10px;
  margin: 25px;
}

button {
  width: 5rem;
}