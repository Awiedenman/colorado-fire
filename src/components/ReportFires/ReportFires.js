import React, { Component } from 'react';
import CurrentFires from '../../containers/CurrentFires/CurrentFires';
import ReportFiresForm from '../../containers/ReportFiresForm/ReportFiresForm';

import './ReportFires.css';

export class ReportFires extends Component {
  render(){
    const style = {
      width: '100%',
      height: '100%'
    };
  
    return (
      <div className='reportFires'>
        <div className='reportFires-map'>
          <CurrentFires
            style={style}/>
          <button className='map-submit'>Submit</button>
        </div>
        <div className='reportFires-form'>
          <ReportFiresForm />
        </div>
      </div>
    );
  }
}

