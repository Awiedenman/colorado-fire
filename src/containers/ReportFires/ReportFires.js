import React, { Component } from 'react'
import CurrentFires from '../CurrentFires/CurrentFires';

import './ReportFires.css';

export class ReportFires extends Component {
  constructor(props){
    super(props);
    this.state = {
      location: ''
    }
  }

  handleChange = (e) =>{
    const { value } =e.target;
    this.setState({location: value})
  }

  render(){
    return(
      <div>
        <div className='reportFires-form'>
          <form>
            <input 
              onChange={this.handleChange} 
              name='location' 
              value={this.state.location}
            />
          </form>
        </div>
        <div className='reportFires-map'>
          <CurrentFires/>
        </div>
      </div>
    )
  }
}

