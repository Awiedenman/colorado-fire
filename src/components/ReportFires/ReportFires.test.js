import React from 'react';
import { ReportFires } from './ReportFires';
import { shallow  } from 'enzyme';

describe('ReportFires', () => {
  test('should match the snapshot', () => {
    const wrapper = shallow(
      <ReportFires/>)

      expect(wrapper).toMatchSnapshot();
  })
  
})
