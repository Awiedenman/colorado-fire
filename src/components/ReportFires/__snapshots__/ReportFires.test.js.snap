// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReportFires should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ReportFires />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="reportFires-map"
>
          <Connect(Wrapper)
                    style={
                              Object {
                                        "height": "100%",
                                        "width": "100%",
                                      }
                    }
          />
          <button
                    className="map-submit"
          >
                    Submit
          </button>
</div>,
        <div
          className="reportFires-form"
>
          <withRouter(Connect(ReportFiresForm)) />
</div>,
      ],
      "className": "reportFires",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <Connect(Wrapper)
              style={
                            Object {
                                          "height": "100%",
                                          "width": "100%",
                                        }
              }
/>,
            <button
              className="map-submit"
>
              Submit
</button>,
          ],
          "className": "reportFires-map",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "style": Object {
                "height": "100%",
                "width": "100%",
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Submit",
              "className": "map-submit",
            },
            "ref": null,
            "rendered": "Submit",
            "type": "button",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <withRouter(Connect(ReportFiresForm)) />,
          "className": "reportFires-form",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="reportFires-map"
>
            <Connect(Wrapper)
                        style={
                                    Object {
                                                "height": "100%",
                                                "width": "100%",
                                              }
                        }
            />
            <button
                        className="map-submit"
            >
                        Submit
            </button>
</div>,
          <div
            className="reportFires-form"
>
            <withRouter(Connect(ReportFiresForm)) />
</div>,
        ],
        "className": "reportFires",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <Connect(Wrapper)
                style={
                                Object {
                                                "height": "100%",
                                                "width": "100%",
                                              }
                }
/>,
              <button
                className="map-submit"
>
                Submit
</button>,
            ],
            "className": "reportFires-map",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "style": Object {
                  "height": "100%",
                  "width": "100%",
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Submit",
                "className": "map-submit",
              },
              "ref": null,
              "rendered": "Submit",
              "type": "button",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <withRouter(Connect(ReportFiresForm)) />,
            "className": "reportFires-form",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
